spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -1009,26 +1009,36 @@ example, to indicate that an LRA should automatically cancel after 100 milliseco
       @HeaderParam(LRA.LRA_HTTP_CONTEXT_HEADER) URI lraId) {...}
 ----
 
-Furthermore, the ability to compensate or complete may be transient capabilities of a
-service so participants can also be timed out. When the time limit is reached
-the LRA is cancelled and participants will be notified via their compensation
-(i.e. the method annotated with `@Compensate`). To set such a time limit add the
-`timeLimit` and `timeUnit` element to the `@Compensate` and `@Complete`
-annotations. If different time limits are set on these two methods then the
-earliest of the two will take effect. An example of how a participant could
-indicate that its' ability to compensate is limited to 100 milliseconds could be:
+When the timeout is reached, the LRA moves to the state `Cancelling` and all registered participants receive a `Compensate`
+call. Just as if the Long Running Action would be closed due to an error.
+
+When a participant joins an existing transaction using the `@LRA(value = LRA.Type.MANDATORY)` annotation, the timeout of
```

Comment:
MANDATORY may be misleading here as this also applies for REQUIRED and SUPPORTS for instance. Can we just say "joins an existing transaction using the `@LRA(...)`" or "joins an existing transaction (for instance using the `@LRA(value = LRA.Type.MANDATORY)`)"?